### ISSUE TITLE
Letter: A11y & design fixes

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#38699]
## Description

While examining the letters app, I found several pages with accessibility issues:

<details><summary>Service not available notice (no <code>h1</code> and content doesn't match design patterns)</summary>

<img width="929" alt="page header plus a sentence then the footer" src="https://user-images.githubusercontent.com/136959/158850821-6f190e38-9df0-4ed8-bbb0-93dedfccd398.png"></details>

<details><summary>Header level problems</summary>

<img width="873" alt="All headers inside the accordion are h3's, the first h3 should have two h4s nested" src="https://user-images.githubusercontent.com/136959/158853649-16b910fc-438b-488f-98a0-09e10880efee.png"></details>

## Tasks

- [ ] Add h1 & alert to no service page
- [ ] Nest h4s under h3 inside benefit summary accordion
- [ ] Update unit tests

## Definition of done

- [ ] All tasks complete
- [ ] All tests passing
